spam/01_spam_tutorial.py

```diff
@@ -192,16 +221,24 @@
 # ### b) Writing an LF to identify spammy comments that use the phrase "check out"
 
 # %% [markdown]
-# Labeling functions in Snorkel are created with the `@labeling_function()` decorator, which wraps a function for evaluating on a single data point (in this case, a row of the DataFrame).
+# Labeling functions in Snorkel are created with the
+# [`@labeling_function()` decorator](https://snorkel.readthedocs.io/en/redux/source/snorkel.labeling.lf.html#snorkel.labeling.lf.core.labeling_function),
+# which wraps a function for evaluating on a single data point.
+# Decorators are a convenient Python mechanism for automatically extending the behavior of the decorated function,
```

Comment:
I think explaining what a decorator is can come before the LF decorator. Something like "we often want to share behaviors among LFs...Python has decorators...we use decorators..." Might be okay to drop python decorator explanation altogether as well

Reply:
we dropped it before but it has come up in every feedback round

Reply:
dropping it again, it's weirdly out of place. keeping the link